src/com/boggle/Main.java

```diff
@@ -6,8 +6,267 @@
 
     public static void main(String[] args) {
 
-        System.out.println(checkString("dog"));
-        System.out.println(checkString("return"));
+       explore(0,0, "");
+
+
+    }
+
+    public static String position(int row, int col){
+        String corner = "corner";
+        String middle = "middle";
+        String side = "side";
+        String actual = "";
+
+        if (row == 0 && col == 0 || row == 0 && col == 4 || row == 4 && col == 0 || row == 4 && col == 4){
+            actual = corner;
+        }
+
+        else if (row == 1 && col == 1 || row == 1 && col == 2 || row == 1 && col == 3){
+            actual = middle;
+        }
+
+        else if (row == 2 && col == 1 || row == 2 && col == 2 || row == 2 && col == 3) {
+            actual = middle;
+        }
+
+        else if (row == 3 && col == 1 || row == 3 && col == 2 || row == 3 && col == 3){
+            actual = middle;
+        }
+
+        else {
+            actual = side;
+
+            if (row == 0 && col == 1 || row == 0 && col == 2 || row == 0 && col == 3){
+                actual += " Upper";
+            }
+            else if (row == 0 && col == 1 || row == 0 && col == 2 || row == 0 && col == 3){
+                actual += " Left";
+            }
+            else if (row == 4 && col == 1 || row == 4 && col == 2 || row == 4 && col == 3){
+                actual += " Right";
+            }
+            else {
+                actual += " Lower";
+            }
+        }
+
+        return actual;
+
+    }
+
+    public static String explore(int row, int col, String path){
+
+        //Created a 5 by 5 Char Array
+        String[][] wordArray = new String[5][5];
```

Comment:
how about iniitializing like this: 
String[][] wordArray = {
            {"a ", "b", "c", "d", "e"},
            {"a ", "b", "c", "d", "e"},
            {"a ", "b", "c", "d", "e"},
            {"a ", "b", "c", "d", "e"},
            {"a ", "b", "c", "d", "e"},
        };

Reply:
will be easier to understand